controllers/reps.js

```diff
@@ -1,9 +1,22 @@
+<<<<<<< HEAD
+const logReps = require('../models/reps');
+=======
 // const log = require('../models/reps');
+>>>>>>> main
 
 // module.exports = {
 //   index,
 // };
 
+<<<<<<< HEAD
```

Comment:
Codacy has a fix for the issue: [Unnecessary semicolon.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7840230)
```suggestion
} 
```